git/git.go

```diff
@@ -573,6 +573,30 @@ func GetCommitSummary(commit string) (*CommitSummary, error) {
 	}
 }
 
+func isCygwin() bool {
+	cmd := subprocess.ExecCommand("uname")
+	out, err := cmd.Output()
+	output := string(out)
```

Comment:
No need to make a temporary variable here, I would just use the `bytes` equivalent:

```go
bytes.Contains(out, []byte("CYGWIN"))
```